apps/beeai-web/tasks.toml

```diff
@@ -0,0 +1,81 @@
+# check
+
+["beeai-web:check"]
+depends = ["beeai-web:check:*"]
+
+["beeai-web:check:prettier"]
+depends = ["setup:pnpm"]
+dir = "{{config_root}}/apps/beeai-ui"
+run = "pnpm prettier --log-level silent --check src"
```

Comment:
Why silent? 😭

(See also other comment about not duplicating tasks with scripts from `package.json`)